mlir/include/mlir/Dialect/Affine/Analysis/Utils.h

```diff
@@ -39,6 +39,7 @@ struct LoopNestStateCollector {
   SmallVector<AffineForOp, 4> forOps;
   SmallVector<Operation *, 4> loadOpInsts;
   SmallVector<Operation *, 4> storeOpInsts;
+  SmallVector<Operation *, 4> memoryEffectOpInsts;
```

Comment:
Missing code comment here on what memory effect ops are. Load and store also have memory effects. Add doc comment for this variable.

Reply:
done